### ISSUE TITLE
MathOptInterface instead of MathProgBase

### ISSUE THREAD

**Issue by joaquimg** [pjssilva/Algencan.jl#15]
Always nice to have new solvers!
I don't know if you intentionally used included MathProgBase (MPB) instead of MathOptInterface (MOI).
In any case MPB works with JuMP up to version 0.18.x, but in order to use the new version of JuMP 0.19 (to be released soon) solvers should use MOI.

**Comment by pjssilva** [comment]
Yes, I know about the new interface. But I had created this interface and it was a kind crude. I decided to learn a little more, move it to Julia 1.0 and create a proper package. I will try to update to MOI in February. Hope it is not difficult.

**Comment by ivborissov** [comment]
Hi, do you still have plans to update the pkg to support MOI (and Windows installation)? It seems the same algorithm is reimplemented in NLopt package (which works well for my NL problems )
https://nlopt.readthedocs.io/en/latest/NLopt_Algorithms/#augmented-lagrangian-algorithm
Looking forward to testing the original implementation

**Comment by pjssilva** [comment]
I am planning to get a new interface once Ernesto makes the new version of Algencan available. I am trying to find a student to help me on this.
